manual/guides/writing/checklist.md

```diff
@@ -30,103 +30,102 @@ As Shakespeare wrote in *Hamlet*, "Brevity is the soul of wit."
+- [ ] The Latin abbreviations cf., et al., or q.v. should not automatically have commas after them.
 - [ ] The abbreviation of *versus* (vs.) should always have a period in American English and is used to contrast things.
-- [ ] You should never say "and etc.," because "et" already means "and" in Latin.
-- [ ] Abbreviations including et (e.g., "et al.") should not have a period after "et" because it is a whole word.
-- [ ] Some abbreviations, but not the majority, need to be capitalized (e.g., "N.B." which means "note well").
-- [ ] Latin (or any Non-English) words, other than et, are usually italicized (e.g., *in situ, in vivo, in vitro,* and *ab initio*).
+- [ ] You should never say "and etc.", because "et" means and in Latin.
+- [ ] Abbreviations including "et" should not have a period after "et" because it is a whole word.
+- [ ] Some abbreviations need capitalization (e.g., "N.B.," which means "note well").
+- [ ] Latin (or any non-English) words other than et are usually italicized (e.g., *in situ*, *in vivo*, *in vitro,* and *ab initio*).
 ```
 
 ## Tables and Figures
 ```markdown
 - [ ] The text should refer to all tables and figures.
-- [ ] When referring to figures by their number, use `Figure 1` and `Table 1.` They should be capitalized and not abbreviated (not `fig. 1` or `figure 1`.)
+- [ ] When referring to figures by their number, use `Figure 1` and `Table 1.` They should be capitalized and not abbreviated (not `fig. 1` or `figure 1`).
 - [ ] Align all columns of numbers in tables such that the decimals line up.
 - [ ] All values should probably have the same number of significant digits in a single column.
 - [ ] Give units for each numerical column.
-- [ ] A table should have only 3 horizontal lines (no vertical lines and no more than 3).
+- [ ] A table should have only three horizontal lines (no vertical lines and no more than three).
 ```
 
 ## Enhancing Math
 ```markdown
-- [ ] Define all variables, with units. If unitless, indicate that this is the case `$[-]$`.
+- [ ] Define all variables with units. If unitless, indicate this is the case `$[-]$`.
 - [ ] Subscripts should be brief and can be avoided with common notation. For example, `$\dot{m}$` is better than `$m_f$` which is superior to `$m_{flow}$`.
 - [ ] Variable names should be symbols rather than words `m` is better than `mass` and `\ksi` is better than `one_time_use_variable`.
 - [ ] The notation `$3.0\times10^{12}$` is preferred over `$3e12$`.
 - [ ] Equations should be part of a sentence.
 - [ ] Equations should be in the `align` environment. Align them at the `=` sign.
-- [ ] Variables should be defined in the `align` environment as well, not buried in paragraphs.
+- [ ] Variables should be defined in the `align` environment, not buried in paragraphs.
 ```
 
 Here's an example of an equation:
 ```latex
-The line is defined as
+The definition of a line is
```

Comment:
This example is referring to a line, not the general definition of lines. I suggest keeping the original text.

Reply:
On second thought, I'm not so sure.

Reply:
Without anything else, I suppose it could go either way; it could be defining a specific line, or a general one.

Reply:
The phrase "... is defined as ..." is so widely used, I wonder if it's exempted from the passive voice rule.

Reply:
Maybe for a line, because it's a common definition, but I think the context we could use "is defined as"
 in our writing has a paper or author we could point to to make it active and not erase that information. All that being said, this example goes against the rule in line 37, so it should be changed from what I put!

Reply:
> Maybe for a line, because it's a common definition, but I think the context we could use "is defined as"
in our writing has a paper or author we could point to to make it active and not erase that information.

Okay I see your point here.

> All that being said, this example goes against the rule in line 37, so it should be changed from what I put!

Applying the rule in line 37 here seems awkward ("definition of a line" -> "line definition"). I'm okay with your original change.